packages/react-components/tests/systems-summary/main-menu-item-state.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { MainMenuItemState } from '../../lib';
+import { itemSummary } from './test.utils';
+import { render } from '@testing-library/react';
+
+test('smoke test', () => {
+  render(<MainMenuItemState itemSummary={itemSummary} onClick={jest.fn()} />);
+});
+
+test('idle and charging panels to be rendered when item value is Robots', () => {
```

Comment:
This seems like a red flag to me, it shouldn't behave differently depending on the "item value".